common-util/functions/index.jsx

```diff
@@ -87,25 +87,25 @@ export const sendTransaction = (fn, account) => sendTransactionFn(fn, account, {
  */
 export const formatToEth = (value, dv = 0) => {
   if (isNil(value)) return dv || 0;
-  return (+ethers.utils.formatEther(value)).toFixed(2);
+  return (+ethers.formatEther(value)).toFixed(2);
```

Comment:
while this is cool, I don't think the `+` is a commonly used for turning strings into numbers, lets keep it readable for others
```suggestion
  return Number(ethers.formatEther(value)).toFixed(2);
```